### PR TITLE
Cura 8562 introduce materials card

### PR DESCRIPTION
**Pull request by jellespijker**
Introduced the _material card_. Which basically only differs from the _plugin card_ with the used icon for certified materials.

I replaced the `isVerified` property for `isCheckedByUltimaker`. Either a plugin is verified by us or not, or a material is certified or not. The status of both the plugin or the material will never be both. Currently the API still returns a `verified` tag for materials. This will in all likelihood be removed once the refactoring is complete.

### CODE REVIEW

File: resources/themes/cura-light/icons/default/Certified.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
...
... 19.8954 12.4876 20.0122 12C19.8933 11.5319 19.6591 11.1011 19.3311 10.7466C18.9836 10.3098 18.6903 9.83249 18.4575 9.3252C18.2639 8.80217 18.1341 8.25771 18.0708 7.7036C18.052 7.22089 17.9126 6.75057 17.6655 6.3355C17.238 6.0731 16.7494 5.92681 16.248 5.9111C15.7105 5.85421 15.1822 5.73055 14.6753 5.543C14.1838 5.31692 13.7224 5.03037 13.3018 4.69C12.9363 4.3467 12.4876 4.10467 12 3.9878ZM15.5361 15.5361L14.1221 14.1211C13.7026 14.5407 13.168 14.8265 12.5861 14.9423C12.0041 15.0581 11.4009 14.9987 10.8526 14.7717C10.3044 14.5447 9.83586 14.1601 9.50619 13.6668C9.17652 13.1734 9.00055 12.5934 9.00055 12C9.00055 11.4066 9.17652 10.8266 9.50619 10.3332C9.83586 9.83986 10.3044 9.45535 10.8526 9.2283C11.4009 9.00126 12.0041 8.94188 12.5861 9.05769C13.168 9.1735 13.7026 9.45928 14.1221 9.8789L15.5361 8.4639C14.8368 7.7648 13.9458 7.28875 12.976 7.09596C12.0061 6.90316 11.0009 7.00227 10.0873 7.38076C9.1738 7.75924 8.393 8.4001 7.84367 9.22232C7.29433 10.0445 7.00113 11.0112 7.00113 12C7.00113 12.9888 7.29433 13.9555 7.84367 14.7777C8.393 15.5999 9.1738 16.2408 10.0873 16.6192C11.0009 16.9977 12.0061 17.0968 12.976 16.904C13.9458 16.7112 14.8368 16.2352 15.5361 15.5361Z" fill="#000E1A"/>
```
Comment (Ghostkeeper) [inline resources/themes/cura-light/icons/default/Certified.svg:2]: The fill colour is incorrect here.

Comment (jellespijker) [reply resources/themes/cura-light/icons/default/Certified.svg:2]: What should be the suggested fill color? This is the icon as provided by UX

File: plugins/Marketplace/resources/qml/PackageCard.qml [hunk 2]
```diff
@@ -83,12 +85,12 @@ Rectangle
...
 
-            enabled: packageData.isVerified
-            visible: packageData.isVerified
+            enabled: packageData.isCheckedByUltimaker
+            visible: packageData.isCheckedByUltimaker
 
             Cura.ToolTip
             {
-                tooltipText: catalog.i18nc("@info", "Verified")
+                tooltipText: packageData.packageType == "plugin" ? catalog.i18nc("@info", "Ultimaker Verified Plugin") : catalog.i18nc("@info", "Ultimaker Certified Material")
```
[the remark this answers is not in the document]
Comment (jellespijker) [reply plugins/Marketplace/resources/qml/PackageCard.qml:93]: Hmmm, neat trick. I never used it as such. Might use it indeed

File: resources/themes/cura-light/icons/default/Certified.svg (hunk 1 above)
Comment (fieldOfView) [reply resources/themes/cura-light/icons/default/Certified.svg]: Icons should not contain colors in the first place. Colors should be applied with UM.RecolorImage.

(Yes, CheckBlueBG.svg is an abomination, in more than one way)

File: plugins/Marketplace/resources/qml/PackageCard.qml (hunk 2 above)
[the remark this answers is not in the document]
Comment (fieldOfView) [reply plugins/Marketplace/resources/qml/PackageCard.qml]: Or, for more readability, just write it out:
```
              tooltipText:
              {
                  if (packageData.packageType == "plugin")
                  {
                      return catalog.i18nc("@info", "Ultimaker Verified Plugin")
                  }
                  return catalog.i18nc("@info", "Ultimaker Certified Material")
```

**Pull request merged**